src/App/PropertyLinks.cpp

```diff
@@ -791,6 +801,17 @@ void PropertyLink::getLinks(std::vector<App::DocumentObject *> &objs,
         objs.push_back(_pcLink);
 }
 
+void PropertyLink::getLinksTo(std::vector<App::ObjectIdentifier>& identifiers,
+                              App::DocumentObject* obj,
+                              const char* subname,
+                              bool all) const
+{
+    (void)subname;
+    if ((all || _pcScope != LinkScope::Hidden) && obj && _pcLink == obj) {

```

Comment:
This if condition is really difficult to parse and understand. Maybe we could use early-exit strategy here, split it into 2-3 ifs with only `return` and proper comments above explaining with this condition is required? Another possibility is to use `bool` variables with proper human readable names to put them in the condition

Reply:
Fair.  Split.